src/main/scala/com/evolutiongaming/cluster/AdaptiveAllocationStrategy.scala

```diff
@@ -55,8 +55,11 @@ class AdaptiveAllocationStrategy(
   override def extractShardId(numberOfShards: Int): ShardRegion.ExtractShardId = {
     case x: ClusterMsg =>
       if (!x.isInstanceOf[PersistenceQuery]) {
-        increment(typeName, x.id)
-        metricRegistry.meter(s"persistence.$typeName.sender.${x.id}.$selfHost").mark()
+        if (!x.isInstanceOf[DoNotCount]) {
+          increment(typeName, x.id)
+          metricRegistry.meter(s"persistence.$typeName.sender.${ x.id }.$selfHost").mark()
+        }
+        metricRegistry.meter(s"persistence.$typeName.command.${ x.id }.${x.getClass.getCanonicalName}.$selfHost").mark()
```

Comment:
Yes, you're right - as we're counting such commands in scope of one persistence entity, there will be no confusion between commands with the same simple name.